tests/run/annot-arg-value-in-java/J.java

```diff
@@ -0,0 +1,8 @@
+public class J {
+  @SuppressWarnings(value = "a")
```

Comment:
Could you add a test with an Int argument instead of a String?

Reply:
Tests with int, bool, and double are added.